### ISSUE TITLE
What is the content of the user help?

### ISSUE THREAD

**Issue by dalf** [searxng/searxng#808]
Related to  #758

What is the content of the user help?

I see:
* the about page
* https://docs.searxng.org/user/search_syntax.html
* https://docs.searxng.org/user/own-instance.html
* may be https://docs.searxng.org/dev/search_api.html
* may be a page about the difference SearXNG and searx

@return42 @not-my-profile @mrpaulblack can you make a list of what you expect for the user help?
It can be section title in a page, a page title, a topic title (topic that cover multiple pages) = there is no need for write down the actual content, just the titles ; the purpose of this issue is to have an overview.

**Comment by not-my-profile** [comment]
I was planning on:

1. https://docs.searxng.org/user/
2. better documentation on categories / how to search / the search syntax

I don't think `/dev/search_api.html` belongs in the user docs. And explaining differences between SearXNG and searx in the README should suffice.

**Comment by return42** [comment]
I think only content that is related to the end user

- "about page"
- "search syntax"

This both parts can (or will) be reorganized.

I think these are not user docs ..

- "Why use a private instance?" --> its more dedicated to devOps
- "Search API" --> not related to an end user
- May be a page about the difference SearXNG and searx --> NO .. but a hint about searx.space might be good.

About the content: please take in mind, its an end user who is looking for some help, contact information and information related to the instance he is using at this moment.

He has its own reason about the instance he uses ("I trust my friend hosting this instance"), there is no need for a comparison with other (meta-) search engines (searx, ddg, ...).

**Comment by not-my-profile** [comment]
>"Why use a private instance?" --> its more dedicated to devOps

I think the user documentation should educate users about trust. So they can choose between:

* continuing to use the current instance
* using a different instance (=> link to searx.space)
* setting up their own instance (=> link to the admin docs)

**Comment by return42** [comment]
> I think the user documentation should educate users about trust.

Hm .. I think 95% of the users I invited to use my instance will not understand anything about .. they just trust me / another reason why thy use my instance: their experience is, that the results of the instance are better than what they get from google !!! :-)

**Comment by not-my-profile** [comment]
Sure but there almost certainly are also users who are just using some random instance without having thought about their trust relationship. I think for such users having something written about trust in the user docs would be nice.

**Comment by return42** [comment]
Other question: will we remove userdoc entirely from https://docs.searxng.org/ ?

If so, what will we do with links to the userdoc e.g. from the README: https://github.com/searxng/searxng/commit/1a210e653c80d33dfebaafc7d6697d7ee58655e1

Will there be any content shared between docs.searxng.org and the userdocs from instance?

**Comment by not-my-profile** [comment]
No we should definitely keep a version of the user docs on `docs.searxng.org`. I'll see how that can be done but I'm currently blocked by #810.

My plan is that `searx/help/<name>.md` is served at `/help/<name>`. So e.g. `searx/help/about.md` will be served at `/help/about`. (I plan on redirecting `/about` to `/help/about` so that old links still work).

Since I want the about page to just be a regular help page I want to get rid of the `about.html` Jinja templates ... in order to do that I however need to incorporate the `aboutextend.html` mechanism -> hence #810.

**Comment by return42** [comment]
> No we should definitely keep a version of the user docs on `docs.searxng.org`. I'll see how that can be done 

( docs --> reST // userdoc --> Markdown ) && shared content --> you see what I mean by "conceptual wrong" ... :confused: .. but let me guess: there is another tool, the myst-parser we will include into our toolchain .. and for what? 

```
Tile
====

lorem ipsum
```

vs.

```
## Title

lorem ipsum
```

And what will we do someone request an image to better illustrate in the userdoc? But lets wait we will find a tool for.

**Comment by not-my-profile** [comment]
No I am not planning on adding further dependencies.

reST and Markdown both become HTML, so integrating them really shouldn't be rocket science ;)

> But lets wait we will find a tool for.

Please spare me your sarcastic comments.

**Comment by HLFH** [comment]
Please merge https://github.com/searxng/searxng/pull/842.

**Comment by return42** [comment]
@dalf I close this issue ... its more or less no longer relevant .. I think we should open individual request (issues) or send a PR when we want to see something in the user documentation ..
